examples/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.3"
+services:
+  tenant-security-proxy:
+    env_file:
+      - ./demo-tsp.conf
+    ports:
+      - 32804:7777
+      - 32805:9000
+    image: gcr.io/ironcore-images/tenant-security-proxy:4.0
```

Comment:
Would it be worth making this just pull the `4` tag? Shouldn't be any breaking changes and could help us update this file less frequently. Just a thought

Reply:
Yep. Done.